notebooks/Emukit-tutorial-sensitivity-montecarlo.ipynb

```diff
@@ -657,9 +658,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python [conda env:emukit]",
```

Comment:
These kernel changes don't look right, and they brake the notebook validation tests. Can you revert them (this one, and two lines below)?

Reply:
I reverted the changes to the previous kernel names. The updated python version is ok or should I revert that one as well?